client/storage/index.js

```diff
@@ -14,14 +14,18 @@ const {promises: fs} = require('fs');
 const path = require('path');
 const FormData = require('form-data');
 const axios = require('axios');
+const config = require('config');
 
 // TODO: for some reason docker fails to resolve module if I move it to another file
-const getDownloadQuery = (chunkId, majorVersion, minorVersion) =>
+// TODO: possibly split this file into several ones after migrating to modules
+const getDownloadQuery = chunkId =>
     gql`{
       transactions(
         tags: [
           {
-            name: "__pn_chunk_${majorVersion}.${minorVersion}_id",
+            name: "__pn_chunk_${config.get(
+        'storage.arweave_experiment_version_major'
```

Comment:
can we add these versions in a variable/const in order not to have to get them from config everytime they are needed? They are used below many times.